azure-pipelines.yml

```diff
@@ -1,894 +1,894 @@
 resources:
-- repo: self
+  - repo: self
 
 trigger:
   batch: true
   branches:
     include:
-    - '*'
+      - '*'
 
 pr:
   branches:
     include:
       - '*'
 
 jobs:
-- job: CheckPullRequest
-  displayName: "Check the Format of Pull Request Title and Content"
-  condition: and(succeeded(), in(variables['System.PullRequest.TargetBranch'], 'dev', 'release'))
-
-  pool:
-    vmImage: 'ubuntu-20.04'
-  steps:
-    - bash: |
-        echo "Check Title of Pull Request: #$(System.PullRequest.PullRequestNumber)"
-        title=$(curl https://api.github.com/repos/$(Build.Repository.Name)/pulls/$(System.PullRequest.PullRequestNumber) | jq -r '.title')
-        echo $title
-        body=$(curl https://api.github.com/repos/$(Build.Repository.Name)/pulls/$(System.PullRequest.PullRequestNumber) | jq -r '.body')
-        echo $body
-        if [ "$(System.PullRequest.TargetBranch)" != "release" ] && echo $title | grep -iqF hotfix:; then
-          echo "Hotfix PR should target release branch."
+  - job: CheckPullRequest
```

Comment:
According to https://yaml.org/spec/1.2.2/#21-collections, collections don't need exetra indent.